modules/compression/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@loaders.gl/compression",
+  "version": "2.3.0-alpha.6",
+  "description": "Decompression and compression plugins for loaders.gl",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/visgl/loaders.gl"
+  },
+  "keywords": [
+    "webgl",
+    "loader",
+    "3d",
+    "mesh",
+    "point cloud"
+  ],
+  "types": "src/index.d.ts",
+  "main": "dist/es5/index.js",
+  "module": "dist/esm/index.js",
+  "esnext": "dist/es6/index.js",
+  "sideEffects": false,
+  "files": [
+    "src",
+    "dist",
+    "README.md"
+  ],
+  "scripts": {
+    "pre-build": "npm run build-bundle && npm run build-bundle -- --env.dev",
+    "build-bundle": "webpack --display=minimal --config ../../scripts/bundle.config.js"
+  },
```

Comment:
If these work on arbitrary ArrayBuffers, which are transferable, would it make sense to tie these up to work on Workers? Or is the idea that compression/decompression would happen within other loaders, where the code is already on a worker?

Reply:
Yes, that would be a good next step, and should not be too hard now that worker utils have moved to loader-utils.

But my focus at this point is to get decompression working for batched mode and we are still not able to do that on workers.